packages/admin/src/pages/build-application/[applicationId]/components/Sections.module.scss

```diff
@@ -6,4 +6,22 @@
 
 .table {
     outline: 1px solid govuk-colour("mid-grey");
-}
\ No newline at end of file
+}
+
+.separator {
+    border-left: 1px solid govuk-colour("mid-grey");
+    padding: 7px;
+    @media (max-width: 640px) {
+        display: none;
+    }
+}   
+
+.b {
```

Comment:
Oops forgot to rename this back to button

Reply:
Fixed in latest commit